src/sdk/model/vac/__json__/ba-backup-repository.ts

```diff
@@ -0,0 +1,17 @@
+export interface BaBackupRepositoryJson {
+  id: string;
+  name: string;
+  repository_uid: string;
+  server_name: string;
+  company_name: string;
+  location_name: string;
+  capacity: number;
+  capacity_units: string;
+  free_space: number;
+  free_space_units: string;
```

Comment:
this field does not exist on the server response

Reply:
Hum, it has been updated lately. I had the old version of it. Thank you for reporting it ! I'll update all of that.